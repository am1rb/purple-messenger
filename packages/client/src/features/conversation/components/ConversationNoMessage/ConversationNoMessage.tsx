import React from "react";

function MessageIsTyping() {
  return <>No message</>;
}

export default MessageIsTyping;
