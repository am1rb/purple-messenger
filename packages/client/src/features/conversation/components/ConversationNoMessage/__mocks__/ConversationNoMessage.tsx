import React from "react";

function conversationNoMessage() {
  return <span data-testid="conversation-no-message" />;
}

export default conversationNoMessage;
