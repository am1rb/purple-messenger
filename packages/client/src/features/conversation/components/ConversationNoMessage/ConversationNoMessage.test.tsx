import React from "react";
import { render } from "@testing-library/react";
import ConversationNoMessage from "./ConversationNoMessage";

describe("The <ConversationNoMessage /> tests", () => {
  it("Should match the snapshot", () => {
    const { container } = render(<ConversationNoMessage />);
    expect(container).toMatchSnapshot();
  });
});
