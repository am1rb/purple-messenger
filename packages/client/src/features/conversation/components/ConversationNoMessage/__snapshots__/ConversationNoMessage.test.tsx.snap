// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The <ConversationNoMessage /> tests Should match the snapshot 1`] = `
<div>
  No message
</div>
`;
